mve_intrinsics/mve.md

```diff
@@ -2,6 +2,7 @@
 title: Arm MVE Intrinsics
 version: 2021Q2
 date-of-issue: 02 July 2021
+landscape: true
 header_counter: true
 toc: true
```

Comment:
These are set to true in all specs, you can remove them from the yaml header and enforce them in the TeX template.

Reply:
These aren't latex variables, but ones used for generating the website.